src/pal/inc/rt/palrt.h

```diff
@@ -809,6 +838,16 @@ enum tagMIMECONTF {
 /******************* shlwapi ************************************/
 
 // note: diff in NULL handing and calling convetion
+#ifdef PAL_STDCPP_COMPAT
```

Comment:
PAL_STDCPP_COMPAT should not be needed here - PAL_STDCPP_COMPAT version should work for everybody.